plugins/enthusiast-source-sanitycms/README.md

```diff
@@ -0,0 +1,23 @@
+# Enthusiast Source Sanity CMS
```

Comment:
Could you add a separate docs page, that explains how to configure it? It would be good to give users clarity what the config parameters stand for.

Something like this one: https://upsidelab.io/tools/enthusiast/docs/plugins/shopify-integration